com.cubrid.cubridmigration.core.testfragment/.classpath

```diff
@@ -1,14 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/J2SE-1.5"/>
-	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src"/>
-	<classpathentry exported="true" kind="lib" path="lib/cglib-nodep-2.1_3.jar" sourcepath="com.cubrid.cubridmigration.core.testfragmentsrc.zip"/>
-	<classpathentry exported="true" kind="lib" path="lib/hamcrest-core-1.1.jar" sourcepath="com.cubrid.cubridmigration.core.testfragmentsrc.zip"/>
-	<classpathentry exported="true" kind="lib" path="lib/hamcrest-library-1.1.jar" sourcepath="com.cubrid.cubridmigration.core.testfragmentsrc.zip"/>
-	<classpathentry exported="true" kind="lib" path="lib/jmock-2.5.1.jar" sourcepath="com.cubrid.cubridmigration.core.testfragmentsrc.zip"/>
-	<classpathentry exported="true" kind="lib" path="lib/jmock-junit4-2.5.1.jar" sourcepath="com.cubrid.cubridmigration.core.testfragmentsrc.zip"/>
-	<classpathentry exported="true" kind="lib" path="lib/jmock-legacy-2.5.1.jar" sourcepath="com.cubrid.cubridmigration.core.testfragmentsrc.zip"/>
-	<classpathentry exported="true" kind="lib" path="lib/objenesis-1.0.jar" sourcepath="com.cubrid.cubridmigration.core.testfragmentsrc.zip"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-17">
+		<attributes>
+			<attribute name="module" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="lib" path="lib/cglib-nodep-2.1_3.jar"/>
+	<classpathentry kind="lib" path="lib/hamcrest-core-1.1.jar"/>
+	<classpathentry kind="lib" path="lib/hamcrest-library-1.1.jar"/>
+	<classpathentry kind="lib" path="lib/jmock-2.5.1.jar"/>
+	<classpathentry kind="lib" path="lib/jmock-junit4-2.5.1.jar"/>
+	<classpathentry kind="lib" path="lib/jmock-legacy-2.5.1.jar"/>
+	<classpathentry kind="lib" path="lib/objenesis-1.0.jar"/>
+	<classpathentry kind="lib" path="C:/Users/rathana/Downloads/slf4j-log4j12-1.6.4.jar"/>
```

Comment:
You referenced the file path C drive in your PC, codes of this style should be changed.

Reply:
Oh, Thank you for this suggestion. 
Yes, It should not referenced to my Pc.
I should copy it to lib folder in project and do  the build path again.